### PR TITLE
Add annotation with proper types for entity properties

### PR DESCRIPTION
**Pull request by bocharsky-bw**
The most IDEs start showing returned types on getter calls after these changes.

### CODE REVIEW

**Comment by bocharsky-bw** [comment]
I also wonder about using `@var` annotation for setters and `@return` one for getters in Symfony Demo. What do you think about adding it?

**Comment by javiereguiluz** [comment]
If we add the `@var` for properties, I guess it makes sense it add it to getters/setters too.

File: src/AppBundle/Entity/Post.php [hunk 1]
```diff
@@ -22,56 +22,74 @@
 class Post
 {
     /**
+     * @var int
```
Comment (stof) [inline src/AppBundle/Entity/Post.php]: useless for constants. The IDE already reads the value anyway. A constant is not a variable

Comment (bocharsky-bw) [inline src/AppBundle/Entity/Post.php]: Reverted.
Thanks, Christophe!

**Comment by xabbuh** [comment]
When the properties are annotated, you shouldn't have to repeat the type for the returned values of the getter methods as IDEs are able to infer the return types then.

**Comment by bocharsky-bw** [comment]
@xabbuh Do you suggest to fully revert commit about annotations for setters / getters?

**Comment by xabbuh** [comment]
@bocharsky-bw For setters where we do not have type hints (i.e. scalar type arguments) we should keep them.

**Pull request closed without merging**